### DISCUSSION TITLE
NPC Sell Prices

### DISCUSSION THREAD

**Discussion by drees5** [HypixelDev/PublicAPI#359] [Q&A]
Is there any way of getting the prices for selling items to the NPC's other than manually checking each item?

If anyone knows a way or has already created a dictionary with the item ID's (same ones returned by the API) and NPC sell prices please tell me before I waste an hour of my time.

Any help is much appreciated.

**Comment by AzureAaron** [comment]
> Is there any way of getting the prices for selling items to the NPC's other than manually checking each item?
> 
> If anyone knows a way or has already created a dictionary with the item ID's (same ones returned by the API) and NPC sell prices please tell me before I waste an hour of my time.
> 
> Any help is much appreciated.

No, the only way to retrieve NPC Sell Prices currently is by manually checking them yourself. I’m not aware of any other method to checking them other than this.
